Change interfaces from MAPMLElement to HTMLElement

Malvoz opened the pull request:
Fix https://github.com/Maps4HTML/MapML/issues/73.

Malvoz committed:
Change interfaces from MAPMLElement to HTMLElement

prushforth commented on spec/index.html:
```diff
@@ -1428,7 +1428,7 @@ <h5>The <code>&lt;<dfn id="the-mapml-element">mapml</dfn>&gt;</code> element</h5
             <pre class="idl">
             [Exposed=Window]
-            interface MAPMLMapMLElement : MAPMLElement {
+            interface HTMLHtmlElement : HTMLElement {
```
I think at the moment this should be "interface HTMLMapmlElement : HTMLElement".  @zcorpan and I did not close the loop on this conversation, but the reasoning is that the <mapml> root document element would still be necessary for a file that is read from a file system and assigned to the browser application by its "filename.ext" file system file type extension mapping, but for which the browser has no previous hint to go by, such as a <!DOCTYPE ...> declaration (In such a case the browser would be expected, as it does with, for example, "myfile.png", synthesize an HTML document enclosing the markup that would render the map layer with default starting conditions, if possible).

Malvoz replied:
Okay, changed interface to `HTMLMapmlElement` in https://github.com/Maps4HTML/MapML/pull/142/commits/0de8500ac997ea223c56df94b6ba9b1998cdf107.

Malvoz replied:
Another thing though, we're currently describing the `map` element as a media element:

> The [`map`](https://maps4html.org/MapML/spec/#the-map-element) element, in conjunction with child [`layer`](https://maps4html.org/MapML/spec/#the-layer-element) and optionally [`area`](https://maps4html.org/MapML/spec/#the-area-element) elements, defines a map [media element](https://html.spec.whatwg.org/multipage/media.html#media-element).

That means the element interface for `<map>` should be "HTMLMapElement : [HTMLMediaElement](https://html.spec.whatwg.org/multipage/media.html#htmlmediaelement)".

But, I am responsible for linking to the "[media element](https://html.spec.whatwg.org/multipage/media.html#media-element)" WHATWG definition, which I first should ask, did you intend to define the map element as a "media element" per that definition (with the presumption that the definition would be extended to include `map`)?

prushforth reviewed (changes requested):
Thanks for doing this!  It will help the professional reviewers avoid confusion, to some extent :-).

Please see the comment I left; apart from that LGTM.

Malvoz committed:
interface HTMLHtmlElement => HTMLMapmlElement

Fix requested change in https://github.com/Maps4HTML/MapML/pull/142#discussion_r517657818

prushforth commented:
@Malvoz no I think the HTMLMediaElement interface would be incorrect for the map element. The use of the term 'media' here refers to MIME media types, at most, although it's not clear thattjeterm should be use in this context. I will review and reword.

Malvoz commented:
@prushforth Okay, sorry for making such an assumption.

Feel free to merge/approve this if you think it's ready, and we can deal with re-wording/remove links to the "media element" definition separately.
